nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/QueryDatabaseTable.java

```diff
@@ -90,6 +90,15 @@
             .addValidator(StandardValidators.NON_NEGATIVE_INTEGER_VALIDATOR)
             .build();
 
+    public static final PropertyDescriptor MAX_ROWS_PER_FLOW_FILE = new PropertyDescriptor.Builder()
+            .name("Max Rows in Flow File")
```

Comment:
Maybe "Max Rows Per Flow File"? minor/cosmetic point but might make it more clear for users. Would need to be changed elsewhere in the file for consistency